clients/ui/bff/internal/models/model_registry.go

```diff
@@ -1,7 +1,8 @@
 package models
 
 type ModelRegistryModel struct {
-	Name        string `json:"name"`
-	DisplayName string `json:"displayName"`
-	Description string `json:"description"`
+	Name          string `json:"name"`
+	DisplayName   string `json:"displayName"`
+	Description   string `json:"description"`
+	ServerAddress string `json:"serverAddress"`
```

Comment:
For the 1:1 with the midstream, we'll also need to include fields for Resource name, Host, Port (is this the same as `ServerAddress` currently?), Username, Password, and Database. Based on the "Edit" modal, we will need to handle these additional fields (shown below).
<img width="817" alt="Screenshot 2025-03-11 at 9 31 40 AM" src="https://github.com/user-attachments/assets/9c932bff-eece-479f-bb35-fc34d1f1da4a" />

To make sure we can pre-fill this data properly in the standalone, could we add these fields to the struct? This would help ensure the standalone works with the full set of fields needed for the modal, and allow us to pre-populate data similarly to the "Details" screens for models and model versions. WDYT @ederign ?

Reply:
@jenny-s51 thank you so much for the review! Those fields would be on :

curl -i -H "kubeflow-userid: [user@example.com](mailto:user@example.com)" "localhost:4000/api/v1/settings/model_registry/test?namespace=kubeflow"

https://gist.github.com/ederign/30b69e8082143e4e31ada2ee606e6da5#file-gistfile1-txt-L28-L40

The model is model_registry_kind.go.

is this what you need?

Reply:
The current ServerAddress on model registry is just a prebuild URL for our info. You should use the endpoitns from /settings/